content/kubermatic/main/architecture/requirements/cluster-requirements/_index.en.md

```diff
@@ -50,27 +50,9 @@ If you have more than one network adapter, and your Kubernetes components are no
 
 ## Check Required Ports
 
-The tables below list the ports that need to be open for communication, e.g. by allowing them via a firewall.
+Please ensure that all nodes in a user cluster can communicate without restriction to ensure functionality of CNI/CSI and Kubernetes itself.
+In addition, user cluster nodes must be able to connect to the Seed clusters nodeport-proxy. This depends on the [expose strategy]({{< ref "../../../tutorials-howtos/networking/expose-strategies" >}})
 
-### Master Cluster Master Node(s)
-
-| Protocol | Direction | Port Range | Purpose                 |
-|----------|-----------|------------|-------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   |
-| TCP      | Inbound   | 2379-2380  | etcd server client API  |
-| TCP      | Inbound   | 10250      | kubelet API             |
-| TCP      | Inbound   | 10251      | kube-scheduler          |
-| TCP      | Inbound   | 10252      | kube-controller-manager |
-| TCP      | Inbound   | 10255      | Read-only kubelet API   |
-
-### Worker Node(s) & User Cluster Worker Nodes
-
-| Protocol | Direction | Port Range  | Purpose               |
-|----------|-----------|-------------|-----------------------|
-| TCP      | Inbound   | 10250       | kubelet API           |
-| TCP      | Inbound   | 10255       | Read-only kubelet API |
-| TCP      | Inbound   | 30000-32767 | NodePort Services**   |
-
-** Default port range for [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service/).
-
-Any port numbers marked with * are overridable, so you will need to ensure any custom ports you provide are also open.
+Tunneling: Port 6443 and 8088 via LoadBalancer service.
+NodePort: By default port 30000-32767 via NodePort service.
+LoadBalancer: By default port 30000-32767 via LoadBalancer service.
```

Comment:
This doesn't look nice in the preview at all. Was it meant to be a list?

I think networking people would generally expect tables here, so I'd personally prefer to bring back the old tables for both master/seeds and user clusters to clarify directions and help translating requirements into firewall rules. I can work on an example of what I mean in a bit.

Reply:
| Source    | Direction  | Destination | Ports               | Purpose                 |
|----------|-----------|------------|-------------- | -------------------|
| KKP Users | Inbound | Master Load Balancer IPs | 443/tcp | Access to KKP Dashboard |
| KKP Users | Inbound | Master Load Balancer IPs | 6443/tcp* | Access to User Cluster API Endpoints (Tunneling Strategy) |
| KKP Users | Inbound | Master Load Balancer IPs | 30000-32767/tcp** | Access to User Cluster API Endpoints (LoadBalancer Strategy) |
| KKP Users | Inbound | Master Nodes | 30000-32767/tcp*** | Access to User Cluster API Endpoints (NodePort Strategy) |
| User Cluster Networks | Inbound | Master Load Balancer IPs | 6443/tcp*, 8088/tcp* | Node access to User Cluster API endpoints and konnectivity server (Tunneling Strategy) |
| User Cluster Networks | Inbound | Master Load Balancer IPs | 30000-32767/tcp** | Node access to User Cluster API endpoints and konnectivity server (LoadBalancer Strategy) |
| User Cluster Networks | Inbound | Master Nodes | 30000-32767/tcp*** | Node access to User Cluster API endpoints and konnectivity server (NodePort Strategy) |

I'm not confident this is the best way to showcase this, but I think something like this would help newcomers a lot to understand what communication from where to where they need to allow.

Reply:
I liked the Idea and added a Table based on your suggestion :)